lib/stats/stats-control.c

```diff
@@ -116,7 +143,7 @@ static void
 control_connection_reset_stats(ControlConnection *cc, GString *command, gpointer user_data, gboolean *cancelled)
 {
   GString *result = g_string_new("OK The statistics of syslog-ng have been reset to 0.");
-  _reset_counters();
+  _reset_counters(NULL, TRUE);
```

Comment:
this should use the public function (as now sometimes we are using the internal one and in other cases the public one) this took me a while to figure out.